tools/generator/rules.py

```diff
@@ -21,14 +25,15 @@ def make_rules(game, out_dir: Path) -> None:
         'game_state.hh'
     ]
     for tpl in base_files:
-        gen.template(tpl)
+        gen.template(os.path.join('src', tpl))
```

Comment:
nit: I'd rather have the list of `base_files` be prefixed by `src/` and include `wscript` so you don't have to special case it and it's more explicit.

Reply:
done